emulator/blockchain.go

```diff
@@ -586,6 +598,8 @@ func configureFVM(blockchain *Blockchain, conf config, blocks *blocks) (*fvm.Vir
 		fvm.WithTransactionFeesEnabled(conf.TransactionFeesEnabled),
 		fvm.WithReusableCadenceRuntimePool(customRuntimePool),
 		fvm.WithEntropyProvider(&dummyEntropyProvider{}),
+		// todo use a configuration flag to enable/disable this
```

Comment:
Isn't this TODO already addressed? The argument that's passed is already the configuration option

Reply:
yes, removed it.